larpix/format/hdf5format.py

```diff
@@ -57,6 +57,118 @@
 The file data is saved in HDF5 datasets, and the specific data format
 depends on the LArPix+HDF5 version.
 
+Version 2.0 description
+^^^^^^^^^^^^^^^^^^^^^^^
+
+For version 2.0, there are two dataset: ``packets`` and ``messages``.
+
+The ``packets`` dataset
+contains a list of all of the packets sent and received during a
+particular time interval.
+
+    - Shape: ``(N,)``, ``N >= 0``
+
+    - Datatype: a compound datatype (called "structured type" in
+      h5py/numpy). Not all fields are relevant for each packet. Unused
+      fields are set to a default value of 0 or the empty string.
+      Keys/fields:
+
+        - ``io_group`` (``u1``/unsigned byte): an id associated with the
+          high-level io group associated with this packet
+
+        - ``io_channel`` (``u1``/unsigned byte): the id associated with the
+          mid-level io channel associated with this packet
+
+        - ``packet_type`` (``u1``/unsigned byte): the packet type code, which
+          can be interpreted according to the map stored in the
+          'packets' attribute 'packet_types'
+
+        - ``chip_id`` (``u1``/unsigned byte): the LArPix chip id
+
+        - ``parity`` (``u1``/unsigned byte): the packet parity bit (0 or
+          1)
+
+        - ``valid_parity`` (``u1``/unsigned byte): 1 if the packet
+          parity is valid (odd), 0 if it is invalid
+
+        - ``downstream_marker`` (``u1``/unsigned byte): a marker to indicate the
+          hydra io network direction for this packet
+
+        - ``channel_id`` (``u1``/unsigned byte): the ASIC channel
+
+        - ``timestamp`` (``u8``/unsigned 8-byte long int): the timestamp
+          associated with the packet. **Caution**: this field does
+          "triple duty" as both the ASIC timestamp in data packets
+          (``type`` == 0), as the global timestamp in timestamp
+          packets (``type`` == 4), and as the message timestamp in
+          message packets (``type`` == 5).
+
+        - ``dataword`` (``u1``/unsigned byte): the ADC data word
+
+        - ``trigger_type`` (``u1``/unsigned byte): the trigger type assciated
+          with this packet
+
+                ('counter','u4'),
+                ('fifo_diagnostics_enabled','u1'),
```

Comment:
These 2 lines look out of place?

Reply:
Yes not sure what happened there, must be a copy / paste issue.